src/quick-lint-js/parse.h

```diff
@@ -3026,6 +3026,23 @@ class parser {
               });
           goto initialize_variable;
 
+        case token_type::kw_typeof:
+        case token_type::kw_class:
+        case token_type::kw_new: {
+          bool has_leading_newline = this->peek().has_leading_newline;
+          if (!has_leading_newline) {
+            const char8 *here = this->lexer_.end_of_previous_token();
+            this->error_reporter_->report(error_missing_equal_after_variable{
+                .expected_equal = source_code_span(here, here),
+            });
+          }
+          this->visit_binding_element(variable, v, declaration_kind);
+          this->skip();
```

Comment:
Why are we skipping the `class` keyword? `class { potato }` is not the same as `{ potato }`.

Reply:
Yes that was wrong I have fixed it now.